### DISCUSSION TITLE
Public chat spam filter feature ?

### DISCUSSION THREAD

**Discussion by johndejohn** [status-im/status-legacy#11830] [Ideas]
# Feature Issue
You can block them, but they also create new account send again...

![Capture+_2021-02-19-01-32-16.png](https://user-images.githubusercontent.com/40263018/108431759-5becd880-7254-11eb-975e-979d55680044.png)

[comment]: # (Please replace ... with your information. Remove < and >)

### User Story
As a <user|developer|...>, I want to <task> **so that** <goal>. 

### Description
[comment]: # (Describe the feature you would like and include designs and Figma links if available.)
[comment]: # (Describe the use cases the implementation should cover. E.g. New users, existing users, public chat, group chat, etc)
_Summary_: ...

### Acceptance Criteria
[comment]: # (Rules for the future PR to be accepted.)

### Notes
[comment]: # (For the developer who will implement. Include planning poker estimates, guidance, etc.)

**Comment by bitsikka** [comment]
I have thought about this problem and would like to note it down here.

### For the current public chat feature
We could perhaps have an opt-in, BrightID verified filter one can activate for a given public chat, to only see BrightID verified user's messages. On top of that if one blocks a BrightID verified user for distasteful speech, the offending user cannot come back again and again to disturb the offended user.

### For upcoming community feature:
This idea is premature at the time of writing - neither BrightID integration, nor Community feature is out yet.

but,

Communities could decide to use BrightID sybil-resistant feature to require members of the community to be BrightID verified. That way, a misbehaving member would need to be blocked only once.

**Comment by johndejohn** [comment]
I think the app bundle with device hardware info, wifi mac address, model number etc.. , will much pratic and easy way to avoid create spam accounts.
And the user, use bundled app, just could create few accounts. max 5 accounts limit something.
Prevent spam chat also need to waku filters

![Capture+_2021-02-19-21-10-15.png](https://user-images.githubusercontent.com/40263018/108544138-19300c80-72f7-11eb-9df8-eee34c5baff6.png)
![Capture+_2021-02-19-21-11-31.png](https://user-images.githubusercontent.com/40263018/108544161-1f25ed80-72f7-11eb-8b8a-3fe33629aef2.png)

**Comment by bitsikka** [comment]
Cool 👍🏽 !

Hadn't thought about it in those ways. Thanks!

There definitely are various ways to prevent reasonable amount and types of spam, perhaps with various tradeoffs.

I'd imagine the tradeoffs would have to be weighted in terms of privacy, security, performance, and practicality; As well as whether it is a global and top-down enforced rule, or peer-to-peer and bottom-up enforcement.

Curious about Waku filters. Will study.
